packages/console/src/pages/Users/components/CreateForm/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { customAlphabet } from 'nanoid';
+
+const alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+const passwordLength = 8;
+
+// Note: password requires a minimum of 8 characters and contains a mix of letters, numbers, and symbols.
+export const createPassword = () => {
+  const randomLength = Math.floor(Math.random() * (passwordLength - 1)) + 1;
```

Comment:
as it's an initial password, i think there's no need to random length

Reply:
and it'll be great to keep it simple, e.g. fixed one number, fixed one alphabet, and followed by six random number or alphabet chars

Reply:
+ 1
${randomDigit}${randomAlphabet}${randomDigitOrAlphabet}(6)

Reply:
updated.